src/mjolnir/osmway.cc

```diff
@@ -173,13 +173,13 @@ OSMWay::GetNames(const std::string& ref, const UniqueNames& name_offset_map, uin
 std::vector<std::string> OSMWay::GetTaggedNames(const UniqueNames& name_offset_map) const {
 
   std::vector<std::string> names;
-  std::vector<std::string> tokens;
 
+  auto encode_tag = [](TaggedName tag) { return std::string(1, static_cast<char>(tag)); };
```

Comment:
im not sure if we should use `char` i think it should be like `std::string::CharT`, i think its platform dependent i guess is my point